src/collections/Grid/GridColumn.js

```diff
@@ -112,4 +113,11 @@ GridColumn.propTypes = {
   width: PropTypes.oneOf(SUI.WIDTHS),
 }
 
+GridColumn.create = createShorthandFactory(
```

Comment:
`createShorthandFactory` accepts only two agruments, in this case correct usage is: 
```js
createShorthandFactory(GridColumn,  children => ({ children }))
```